migration-tools/migrate-rosdistro.py

```diff
@@ -18,7 +18,7 @@
 
 # These functions are adapted from Bloom's internal 'get_tracks_dict_raw' and
 # 'write_tracks_dict_raw' functions.  We cannot use them directly since they
-# make assertions about the release repository that are not true during the
+# make assumptions about the release repository that are not true during the
```

Comment:
For what its worth, I do believe you can "make assertions"; that is, you are asserting that something is true.

That being said, I also believe that the author of this comment probably meant "make assumptions", so I'm good with this change.

Reply:
> I do believe you can "make assertions"; that is, you are asserting that something is true.

true. People can "make assertions", but this is code, so it sounded weird to me.

Reply:
In this case bloom does indeed *assert* that there is only one remote with [check_for_multiple_remotes](https://github.com/ros-infrastructure/bloom/blob/9e81373b0e9be8d0e154dcc57ec8530b4ddc8ea5/bloom/config.py#L283-L294) and issues a fatal error.

Since the purpose of the comment is to explain and the explanation was insufficient, we should not revert to the previous comment but the merged state of this PR is also incomplete. The code does not make an assumption, it enforces a constraint which produces an unrecoverable error if violated.

In the interest not of advocating for the previous state but of describing how I chose those words: I have in the past described any code that uses an `assert`-style function or generating an unrecoverable error based on a checked condition as code that "makes an assertion" or code that "asserts a condition".

@gavanderhoorn does the following text make the original intent clearer?
```
We cannot use them directly since they require the release repository to have exactly one remote`
```

Reply:
That's fine by me.

> In this case bloom does indeed *assert* [..]

yes, the code asserts. It does not *make assertions* ;)

At least to me, the first sounds ok (as in: the code makes sure that ..), the second does not sound ok.

But I'm not a native speaker, and we're almost bike-shedding here.

Reply:
I interpret both phrases as synonymous and over a cold beverage would love to discuss personification of code in comments some time. However my overriding concern is clarity having been shown that one phrasing causes confusion I have no problem adapting.